OneCommunityGlobal/HGNRest
# Luis add email notifcation to warning tracker backend

# Description
Backend for sending an email, 
Notable changes were, adding new functions to send the email, adjusted the filter method, and added helper functions to get the admin of the team .

## Related PRS (if any):
To test this backend PR you need to checkout the #[2752](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2752) frontend PR.
…

## Main changes explained:
--created helper functions to send an email, but filtering and fetching the admin of the team.
--adjust filtering to check if an email should or shouldn't be sent 


## How to test:
1. check into current branch
2. do `npm run build && npm run start`
3. follow steps of the frontend PR #[2752](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2752) 

## Note:
Ensure you follow the steps of the Frontend PR and Ivy Chen's PR.
